internal/views/feeder.go

```diff
@@ -80,11 +80,26 @@ func (r *root) enter(row int) (feeder, error) {
 		return nil, errNoData
 	}
 	r.enteredAt = row
-	return newTopic(r.cli, r.topics[row], r.width, r.height, r.flashMessage)
+	return newTopic(r.cli, r.topics[row + r.height * r.pg], r.width, r.height, r.flashMessage)
```

Comment:
If you revert this line back to the original along with my other suggested changes this PR seems to work.